hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/signature/StringToSignProducer.java

```diff
@@ -288,22 +309,14 @@ static void validateSignedHeader(
       String header,
       String headerValue
   )
-      throws OS3Exception {
+      throws OS3Exception, DateTimeParseException {
     switch (header) {
     case HOST:
-      try {
-        URI hostUri = new URI(schema + "://" + headerValue);
-        InetAddress.getByName(hostUri.getHost());
-        // TODO: Validate if current request is coming from same host.
-      } catch (UnknownHostException | URISyntaxException e) {
-        LOG.error("Host value mentioned in signed header is not valid. " +
-            "Host:{}", headerValue);
-        throw S3_AUTHINFO_CREATION_ERROR;
-      }
+      // TODO: Placeholder for any host validations.
```

Comment:
What type of host validatation are we going to add here now that the old logic is removed?

Reply:
I don't think we need a host validation per se

Reply:
Agreed. Shall we remove this TODO then? @SaketaChalamchala

Reply:
Sure